addons/html_builder/static/src/website_builder/plugins/theme_tab.js

```diff
@@ -1,5 +1,5 @@
 import { Component, useState } from "@odoo/owl";
```

Comment:
hmm, if we move the theme tab here, we should find a way to make it configurable somehow. massmailing will need a "Design" tab